Library/RemoteConfigClientType.swift

```diff
@@ -0,0 +1,35 @@
+import FirebaseRemoteConfig
+import Foundation
+
+public enum RemoteConfigClientTypeError: Error {
+  case activate(_: Error?)
+  case fetch(_: Error?)
+
+  var localizedDescription: String {
+    switch self {
+    case let .activate(error):
+      return "Error activating Remote Config \(error?.localizedDescription ?? "")"
+    case let .fetch(error):
+      return "Error fetching Remote Config \(error?.localizedDescription ?? "")"
+    }
+  }
+}
```

Comment:
Can remove because we don't use/need the error.
We can add this back in later if required.